src/EnergyPlus/PlantChillers.hh

```diff
@@ -268,6 +394,9 @@ namespace PlantChillers {
 		int HRLoopSideNum; // heat recovery water plant loop side index
 		int HRBranchNum; // heat recovery water plant loop branch index
 		int HRCompNum; // heat recovery water plant loop component index
+		bool MyFlag;
+		bool MyEnvrnFlag;
+		ElectricReportVars reports;
```

Comment:
Yeah, this is intermediate.  I was going to drop them all right in, but then I hit a second level of shadowing, and I got nervous about just throwing them in there.  I'm resolving some diffs right now, and once I get those gone, I'm going back through and cleaning this type of stuff up.  Oh, and adding virtual destructors.  Oh, and ...